soh/soh/GameMenuBar.cpp

```diff
@@ -407,6 +407,8 @@ namespace GameMenuBar {
                     UIWidgets::Tooltip("Helps FW persist between ages, gives child and adult separate FW points, and can be used in more places.");
                     UIWidgets::PaddedEnhancementCheckbox("Static Explosion Radius", "gStaticExplosionRadius", true, false);
                     UIWidgets::Tooltip("Explosions are now a static size, like in Majora's Mask and OoT3D. Makes bombchu hovering much easier.");
+                    UIWidgets::PaddedEnhancementCheckbox("Disable First-Person Bombchus", "gDisableFirstPersonChus", true, false);
```

Comment:
I think "Disable First-Person Bombchus" is a confusing name for this. It's not disabling first person chus, it's preventing *forced*/*automatic* first person when using chus.

Not sure what a short way to communicate that would be

Reply:
"Prevent Bombchu Forcing First-Person" or "Bombchus Force First-Person" (on by default)

Reply:
I like "Bombchus Force First-Person" (on by default. It'd probably be good to also add a tooltip explaining that forcing first person is authentic behavior

Reply:
Personally leaning towards "Prevent Bombchu Forcing First-Person", disabled by default. It would otherwise be the only enhancement checkbox enabled by default, which imo could be confusing to people who maybe go through disabling everything for a vanilla experience or enabling everything for a fully modded experience, and perhaps dont look too hard at what they're doing. Makes the tooltip require a little less explanation too as it is clearer which is the original n64 behaviour. Also when I tried having it enabled by default the tooltip stopped displaying and I have no idea why lmao

Reply:
that works for me, i was only really leaning the other way to avoid having really long names, but it seems reasonable enough